pom.xml

```diff
@@ -20,8 +20,9 @@
 
   <properties>
     <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-    <jenkins.version>2.190.3</jenkins.version>
-    <slf4j.version>1.7.26</slf4j.version>
+    <jenkins.version>2.263.1</jenkins.version>
```

Comment:
Why should the plugin no longer support older versions of Jenkins?

Reply:
The jaxb plugin requires at least this version to work.